cfme/automate/provisioning_dialogs.py

```diff
@@ -182,7 +156,7 @@ def update(self, updates, cancel=False, reset=False):
             btn = view.form.save
 
         btn.click()
-        view = self.create_view(ProvDiagDetailsView)
+        view = self.create_view(ProvDiagDetailsView, override=updates)
         # TODO use override in create_view in order to assert view.is_displayed
```

Comment:
**REQUIRED:** This is no longer a todo.   Might want to still have a comment on why you wanted to override updates.